### PR TITLE
Fix wrong import

### PR DESCRIPTION
**Pull request by omatvejev**
## Description

Typo in ioutil
______

For contributor use:

- [x] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-go-sdk/blob/master/CONTRIBUTING.md#styleguides).
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: docs/index.mdx [hunk 1]
```diff
@@ -529,14 +529,14 @@ transaction(greeting: String) {
 ```go
 import (
   "context"
-  "ioutil"
+  "io/ioutil"
```
Comment (turbolent) [inline docs/index.mdx:532]:
```suggestion
  "os"
```

File: docs/index.mdx [hunk 2]
```diff
@@ -529,14 +529,14 @@ transaction(greeting: String) {
...
   "github.com/onflow/flow-go-sdk"
   "github.com/onflow/flow-go-sdk/client"
 )
 
 func main() {
 
-  greeting, err := outil.ReadFile("Greeting2.cdc")
+  greeting, err := ioutil.ReadFile("Greeting2.cdc")
```
Comment (turbolent) [inline docs/index.mdx:539]:
```suggestion
  greeting, err := os.ReadFile("Greeting2.cdc")
```

**Review by turbolent — COMMENTED** [review]
Good catch!

`ioutil` has been deprecated since Go 1.16, so the example should probably be switched to `os` while at it

**Comment by omatvejev** [comment]
👍

**Review by turbolent — APPROVED** [review]

**Pull request merged**